docs/app/pages/components/components-sections/badge/badge/badge.component.html

```diff
@@ -0,0 +1,69 @@
+<p>Demo here</p>
+
+<hr>
+
+<p>
+    The <code>ux-badge</code> directive can be used to easily attached a badge to any DOM element to be used as a label/tag, or to show (x) number of new items relating to it etc.
+</p>
+
+<p>The appearance and behavior of the directive can be configured using the following attributes:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="badgeBackgroundColor" type="string">
+        The color you want to use for the background color. This should be a string value that matches one of the colors available in the <code>ColorService</code>.
+    </tr>
+    <tr uxd-api-property name="badgeTextColor" type="string">
```

Comment:
I'm not sure if this input is required. In all of Roland's designs the text color is either black or white, whichever has the best contrast ratio. In which case we can use our contrast ratio directive to automatically set the colour. I notice this is what Angular Material do, they just have a `matBadgeColor` input which sets the background colour and the text is automatically coloured to the most accessible colour. Thoughts?

Reply:
Yeah we were talking about this primarily as a rarely-used override in case someone really wanted red text or something, but that might be me overthinking it. I'll link this to Roland and see if he can think of any valid use case of text color other than black or white.

Reply:
Just heard from Roland, he thinks it's unlikely that someone would need to override the text color in any situation, so we can go ahead and drop this input.